src/senaite/core/upgrade/v02_07_000.py

```diff
@@ -92,3 +94,16 @@ def mark_invalidated_samples(tool):
         sample._p_deactivate()
 
     logger.info("Mark invalidated samples as IInvalidated [DONE]")
+
+
+@upgradestep(product, version)
+def upgrade_catalog_modified_index(tool):
+    """Update modified index in catalog
+    """
+    logger.info("Upgrade catalog modified index ...")
+    portal = api.get_portal()
+
+    setup_catalog_mappings(portal)
```

Comment:
There is no need to call `setup_catalog_mappings` because no new content type has been added to map against existing catalogs